code/game/objects/structures/safe.dm

```diff
@@ -2,80 +2,110 @@
 CONTAINS:
 SAFES
 FLOOR SAFES
-Safe Codes
-Safe Internals
+SAFE INTERNALS
+SAFE CODES
 */
 
 GLOBAL_LIST_EMPTY(safes)
 
 //SAFES
 /obj/structure/safe
 	name = "\improper Safe"
-	desc = "A huge chunk of metal with a dial embedded in it. Fine print on the dial reads \"Scarborough Arms - 2 tumbler safe, guaranteed thermite resistant, explosion resistant, and assistant resistant.\""
+	desc = "A huge chunk of metal with a dial embedded in it. Fine print on the dial reads \"Scarborough Arms tumbler safe, guaranteed thermite resistant, explosion resistant, and assistant resistant.\""
 	icon = 'icons/obj/structures.dmi'
 	icon_state = "safe"
-	anchored = 1
-	density = 1
-	var/open = FALSE	//is the safe open?
+	
+	anchored = TRUE
+	density = TRUE
+	resistance_flags = LAVA_PROOF | FIRE_PROOF

```

Comment:
Keep in mind our current fire system does not use resistance flags; it goes by `burn_state`

Reply:
The safe shouldn't actually burn thankfully. This is just in case we ever finish porting the damage system.

Reply:
Really, to be honest, we could probably go ahead and kill off `burn_state` and just use `resistance_flags` instead; we wouldn't have to have the object damaging portion for it to work and be checked for; if we ever wanted objects to damage in the same way TG does, then this would be great.